WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailLikesView.swift

```diff
@@ -4,132 +4,73 @@ protocol ReaderDetailLikesViewDelegate: AnyObject {
     func didTapLikesView()
 }
 
-class ReaderDetailLikesView: UIView, NibLoadable {
-
-    @IBOutlet weak var avatarStackView: UIStackView!
-    @IBOutlet weak var summaryLabel: UILabel!
-
-    // Theming support
+final class ReaderDetailLikesView: UIView, NibLoadable {
+    @IBOutlet private weak var avatarStackView: UIStackView!
+    @IBOutlet private weak var summaryLabel: UILabel!
+    @IBOutlet private weak var selfAvatarImageView: CircularImageView!
 
     var displaySetting: ReaderDisplaySetting = .standard {
         didSet {
             applyStyles()
-            updateSummaryLabel()
+            if let viewModel {
+                configure(with: viewModel)
+            }
         }
     }
 
     private var preferredBorderColor: UIColor {
         displaySetting.color == .system ? .systemBackground : displaySetting.color.background
     }
 
-    /// The UIImageView used to display the current user's avatar image. This view is hidden by default.
-    @IBOutlet private weak var selfAvatarImageView: CircularImageView!
-
     static let maxAvatarsDisplayed = 5
-    weak var delegate: ReaderDetailLikesViewDelegate?
-
-    /// Stores the number of total likes _without_ adding the like from self.
-    private var totalLikes: Int = 0
 
-    /// Convenience property that adds up the total likes and self like for display purposes.
-    var totalLikesForDisplay: Int {
-        return displaysSelfAvatar ? totalLikes + 1 : totalLikes
```

Comment:
`displaysSelfAvatar` was the root cause, as the `selfAvatarImageView` `hidden` value was being updated _after_ showing the a number of likes. So, there was an easier fix, but I opted to refactor it to make it completely reliable as there could be other issues.